roles/yoda_external_user_service/tasks/main.yml

```diff
@@ -32,23 +69,42 @@
   ansible.builtin.git:
     repo: "https://github.com/UtrechtUniversity/yoda-external-user-service.git"
     dest: "/var/www/extuser/yoda-external-user-service"
-    version: "{{ yoda_version }}"
+    version: "{{ yoda_eus_version }}"
+    force: true
 
 
-- name: Link public directories of Yoda External User Service
-  ansible.builtin.file:
-    src: "/var/www/extuser/yoda-external-user-service/{{ item }}"
-    dest: "/var/www/extuser/{{ item }}"
-    state: link
-  when: not ansible_check_mode
-  with_items:
-    - public
-    - api
+- name: Check out Yoda portal for themes used by External User Service
```

Comment:
I have tested to what extent a sparse checkout improves performance or required diskspace in this situation. The sparse checkout was not faster than a regular checkout, and required diskspace was only slightly less (roughly 10% of repository size / 3 MB). I think it's best to do a regular checkout for the time being, in this case, since it enables us to use ansible.builtin.git and does not result in a severe performance/diskspace impact.